### PR TITLE
Update to reflect Globus Compute re-branding

### PR DESCRIPTION
**Pull request by rjmello**
[sc-22256]

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
+Using Globus Compute with Binder
 =======================
 
 The easiest way to run these tutorials is via Binder. Using Binder you can run the tutorial notebooks in your browser without installing any code locally.
 
 `Start Binder <https://mybinder.org/v2/gh/funcx-faas/examples/HEAD?filepath=notebooks%2FIntroduction.ipynb>`_
 
-Using funcX locally
+Using Globus Compute locally
 ===================
```
Comment (LeiGlobus) [inline README.rst]: I think you commented on one of my previous PRs lengthening the === :)

File: notebooks/Advanced-FuncXClient.ipynb [hunk 2]
```diff
@@ -146,18 +151,19 @@
...
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Functions with dependencies\n",
     "\n",
-    "funcX requires that functions explictly state all dependencies within the function body. It also assumes that the dependent libraries are available on the endpoint in which the function will execute.  For example, in the following function, we explictly import the time module. "
+    "Globus Compute requires that functions explictly state all dependencies within the function body. It also assumes that the dependent libraries are available on the endpoint in which the function will execute.  For example, in the following function, we explictly import the time module. "
```
Comment (LeiGlobus) [inline notebooks/Advanced-FuncXClient.ipynb]: If you were to make any changes, might as well remove the extra space at the end.

**Review by LeiGlobus — APPROVED** [review]
Looks good.  Comments are both non-functional suggestions so not really necessary.

File: README.rst [hunk 3]
```diff
@@ -1,41 +1,41 @@
...
 
-To install funcX, please ensure you have python3.6+.::
+To install Globus Compute, please ensure you have python3.6+.::
```
Comment (khk-globus) [inline README.rst]: While we're changing this line, you might consider also updating the now out-of-date 3.6.  That's 3.7+ now.  (And, given the EOL in 2.5 months, will soon enough be 3.8+.  But that's not for today.)

**Review by khk-globus — APPROVED** [review]
Same observation as Lei: LGTM and EoL whitespace.

For example:

- `"...blah blah. \n"`
- `"...blah blah. "`

And so forth.  There are number of lines like this.  Meh.  Those of us who sometimes copy and paste from the web dislike those extra spaces we keep finding in our local documents.  But it's also a cross we're already accustomed to bearing so &hellip;

File: README.rst (hunk 1 above)
Comment (rjmello) [reply README.rst]: Sphinx will give a warning for these, so good call.

File: README.rst (hunk 3 above)
Comment (rjmello) [reply README.rst]: Good call.

File: notebooks/Advanced-FuncXClient.ipynb (hunk 2 above)
Comment (rjmello) [reply notebooks/Advanced-FuncXClient.ipynb]: Should be good now.

**Review by khk-globus — APPROVED** [review]
Awesome.  Thanks for this.

**Pull request merged**